### DISCUSSION TITLE
Release v1.0.0

### DISCUSSION THREAD

**Discussion by ChenglongMa** [ChenglongMa/LOLauncher#2] [General]
请参考 [README 文档](https://github.com/ChenglongMa/LOLauncher?tab=readme-ov-file) 使用

<hr /><em>This discussion was created from the release <a href='https://github.com/ChenglongMa/LOLauncher/releases/tag/1.0.0'>Release v1.0.0</a>.</em>

**Comment by Ptou17** [comment]
好像失效了大哥

**Reply by ChenglongMa** [reply]
啊，这么快啊，我现在手头没电脑，最快等我周一更新一下哈，多谢关注！

**Reply by ChenglongMa** [reply]
@Ptou17 你好，我刚刚试了一下还是可以用的啊？游戏外的界面可能是其他语言的，但是你进游戏里也不是中文吗？

以下是我的截图：

![League of Legends (TM) Client 20_05_2024 11_00_21](https://github.com/ChenglongMa/LOLauncher/assets/33501679/9b178175-2dcd-41f4-be6a-77a4344dec84)

**Reply by ChenglongMa** [reply]
@Ptou17 你好，我更新了新的版本，之前的bug改好了，欢迎更新使用！

**Reply by Ptou17** [reply]
牛逼大佬！！！

**Reply by ChenglongMa** [reply]
> 牛逼大佬！！！

多谢😂！
